physionet-django/notification/utility.py

```diff
@@ -292,21 +292,21 @@ def copyedit_complete_notify(request, project, copyedit_log):
     """
     subject = 'Your approval needed to publish: {0}'.format(project.title)
 
-    for email, name in project.author_contact_info():
-        body = loader.render_to_string(
-            'notification/email/copyedit_complete_notify.html', {
-                'name': name,
-                'project': project,
-                'copyedit_log': copyedit_log,
-                'domain': get_current_site(request),
-                'url_prefix': get_url_prefix(request),
-                'signature': email_signature(),
-                'project_info': email_project_info(project),
-                'footer': email_footer()
-            })
-
-        send_mail(subject, body, settings.DEFAULT_FROM_EMAIL,
-                  [email], fail_silently=False)
+    for person in project.author_list():
```

Comment:
please could you briefly explain the need for changing this chunk? if someone has approved, and then we reopen and edit, shouldn't they be required to approve again?

Reply:
In the previos code, the message would be sent to ALL of the authors. I converted this to work as a reminder and its initial function to notify all authors for the first time.

In the reminder part, I don't want to SPAM the author that has already approved, hence I check first if he approved, and if he did not then I continue with the email.